app/routes/index.js

```diff
@@ -32,8 +32,12 @@ export default Ember.Route.extend(SetPageTitle,GetFutureRuns, {
     });
   },
 
-  afterModel() {
-    var channel = this.modelFor('application').channel;
-    this.setTitle(channel.get('name'));
-  }
+	afterModel() {
+		var channel = this.modelFor('application').channel;
+		let name = channel.get('publicSite.siteName');
```

Comment:
`let name = channel.get('publicSite.siteName') || channel.get('name');` would take care of an undefined or null state